.github/container/nsys_jax/nsys_jax/analyses/communication.py

```diff
@@ -95,6 +96,29 @@ def format_bandwidth(data, collective):
             )
         )
 
+    collective_types = set()
+    summary_data = defaultdict(dict)
+    for collective, df in steady_state.communication.groupby(["Collective"]):
+        collective_types.add(collective)
+        summary_data[collective] = df["DurHiddenMsToDurMs"].mean()
+
+    table = PrettyTable()
+    table.field_names = ["Collective", "Mean HiddenToTotalMs"]
+
+    for collective in collective_types:
+        mean_value = summary_data[collective]
+        table.add_row([collective[0], mean_value])
+    print(table)
```

Comment:
The result is visible in the CI here: https://github.com/NVIDIA/JAX-Toolbox/actions/runs/12753978814/job/35547441911#step:13:301
But admittedly that test is not expected to have overlap, so it's not very interesting.

It's also visible in the CI here: https://github.com/NVIDIA/JAX-Toolbox/actions/runs/12753978740/job/35546782084#step:5:104, which is a bit more interesting because the test data (which are committed to the repo for that test, not generated on the fly) do contain some overlap.